src/utils/collection.ts

```diff
@@ -69,6 +69,25 @@ const formatConsentValues = (publisherConsents: Consent[] | null = [], customerC
   if (!publisherConsents || !customerConsents) {
     return {};
   }
+
+  const values: Record<string, string | boolean> = {};
+
+  publisherConsents?.forEach((publisherConsent) => {
+    const consent = customerConsents?.find((customerConsent) => customerConsent.name === publisherConsent.name);
+
+    if (consent) {
+      const value = publisherConsent.provider === 'cleeng' ? consent.state === 'accepted' : consent.value ?? '';
```

Comment:
can we have a more generic solution here? The idea is to not include the provider's name

Reply:
Done, I replaced `provider: 'cleeng' | 'jwp'` with the more generic `isCustomRegisterField: boolean`.